File: src/217.contains-duplicate.scala
/*
 * @lc app=leetcode id=217 lang=scala
 *
 * [217] Contains Duplicate
 */

// @lc code=start
object SolutionLACADSAdAA {
    def containsDuplicate(nums: Array[Int]): Boolean = {
        //this solution uses set theory, a set can have no duiplate element.
        nums.toSet.size != nums.size;
            
    }
}
// @lc code=end

File: src/217.contains-duplicate.java
/*
 * @lc app=leetcode id=217 lang=java
 *
 * [217] Contains Duplicate
 */

// @lc code=start

import java.util.Arrays;

class Solutiona {
    public boolean containsDuplicate(int[] nums) {


      Arrays.sort(nums);
            for (int j =  0; j < nums.length-1; j++) {
                if(nums[j] == nums[j+1]){
                    return true;
                }
            }
       

    return false;
        
    }
}
// @lc code=end

